### PR TITLE
Chatroom model

### PR DESCRIPTION
**Pull request by bakuwaku-cloud**
crteated chatroom model and tags
updated user and message fields

### CODE REVIEW

File: server/models/chatroom.js [hunk 1]
```diff
@@ -1,30 +1,27 @@
-const mongoose = require("mongoose");
-const User = require("./user");
-const Message = require("./message");
+const mongoose = require('mongoose');
```
Comment (stevendreed) [inline server/models/chatroom.js:1]: @astro0725 @PaBoba we keep flipping between `''` and `""`. We should pick one of these conventions and stick with it

File: server/models/chatroom.js [hunk 2]
```diff
@@ -1,30 +1,27 @@
...
+  tags: [{
+    type: ObjectId,
+    ref: 'tag'
+  }],
+  icon: String,
+  activeUsers: [{
+    type: ObjectId,
+    ref: 'user'
+  }],
+  createdAt: {
```
Comment (stevendreed) [inline server/models/chatroom.js:20]: Really clever- I like this

File: server/models/chatroom.js (hunk 1 above)
Comment (bakuwaku-cloud) [reply server/models/chatroom.js:1]: @PaBoba lmk what u prefer

File: server/models/chatroom.js [hunk 3]
```diff
@@ -1,30 +1,27 @@
...
+    type: Date,
+    default: Date.now,
+    expires: 3600
```
Comment (stevendreed) [inline server/models/chatroom.js:23]: yep, this is the core of the application

**Review by stevendreed — APPROVED** [review]
LGTM, really clever work with `Date.Now()`

**Pull request merged**